Project-Lily/Project-Lily/DogListViewController.swift

```diff
@@ -12,11 +12,12 @@ class DogListViewController: UIViewController {
     
     //MARK: - Properties
     
-    var dogDatabase = DogDatabase()
+    //var dogDatabase = DogDatabase()
     var filteredDogArray = [Dog]()
     var selectedIndex: Int?
     var isSearching = false
-   
+    var dogListVM = DogListViewModel()
```

Comment:
This could be overly pedantic, but it makes for good conversation :) - I haven't seen a lot of examples of shortening `ViewModel` to `VM` but I have seen a var like this simply be called `viewModel` or a more long-winded `dogListViewModel`. I think a ViewController can expect to have 1 and only 1 viewModel, so there's less concern for distinguishing between which viewModel it might be (it would be strange for a ViewController to have a dogListViewModel and a birdListViewModel, but that would be a good use case to need to distinguish between them with more verbose names). 

No changes needed here, just food for thought on how we're naming this.

Reply:
Ah ok that is a really good point. I think I'll elongate the name to viewModel instead of vm. I like being specific with dogList because it corresponds to the DogListViewController. Just makes it easier for me to know which viewModel corresponds to a specific viewController

Reply:
sounds good!